packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -290,6 +290,8 @@ export default async function initSanity(
     print('')
   }
 
+  const isNextJs = detectedFramework?.slug === 'nextjs'
+
   const flags = await prepareFlags()
```

Comment:
Need to set `isNextJs` before calling `prepareFlags` because we are using the value in the function. This is not great, but the function already similarly depends on other variables so this solution was deemed the least intrusive.